roles/nova-data/tasks/libvirt.yml

```diff
@@ -12,7 +22,7 @@
     - libvirt-bin={{ nova.libvirt_bin_version }}
     - python-libvirt={{ nova.python_libvirt_version }}
     - qemu-kvm={{ nova.qemu_kvm_version }}
-    - qemu-system-x86={{ nova.qemu_kvm_version }}
+    - qemu-system-{{ fact_arch }}={{ nova.qemu_kvm_version }}
```

Comment:
Actually I think we can make this even easier by just installing `qemu-system` instead of a specific arch.

It'll bring in _all_ the arch system packages, but that's fine, we'll get the ones that will be used and we don't have to play guessing games based on the host arch.